src/assets/locales/fr.json

```diff
@@ -217,74 +244,80 @@
   "media-tile.action.edit-avatar": "Éditer l'avatar",
   "media-tile.action.edit-scene": "Éditer la scène",
-  "media-tile.action.room-info": "Info de la Salle",
+  "media-tile.action.room-info": "Info de la salle",
   "media-tile.action.show-similar-avatars": "Montrer les avatars similaires",
-  "media-tile.joined-room": "Joint {relativeTime}",
+  "media-tile.joined-room": "A rejoint {relativeTime}",
   "mic-permissions-modal.grant-mic-permissions": "Accorder les autorisations au micro",
   "mic-permissions-modal.mic-access-needed": "Vous avez besoin d'un micro pour être entendu par les autres participants.{linebreak}Vous pourrez couper votre micro dans la salle.",
   "mic-permissions-modal.title": "Entrer par un appareil",
   "mic-setup-modal.check-mic": "Vérifiez votre micro et l'audio avant d'entrer.",
-  "mic-setup-modal.enter-room-button": "Entrer dans la Salle",
+  "mic-setup-modal.enter-room-button": "Entrer dans la salle",
   "mic-setup-modal.mic-disabled": "Micro Désactivé",
   "mic-setup-modal.mute-mic-toggle": "Couper mon micro",
   "mic-setup-modal.test-audio": "Cliquer pour Tester l'Audio",
   "mic-setup-modal.test-mic": "Parler pour Tester le micro",
   "mic-setup-modal.title": "Réglages du micro",
   "more-menu-popover.title": "Plus",
-  "more-menu.close-room": "Fermer la Salle",
+  "more-menu.close-room": "Fermer la salle",
   "more-menu.community": "Communauté",
-  "more-menu.controls": "Controles",
-  "more-menu.create-room": "Créer une Salle",
-  "more-menu.enter-leave-room": "Quitter la Salle",
+  "more-menu.controls": "Contrôles",
+  "more-menu.create-room": "Créer une salle",
+  "more-menu.enter-leave-room": "Quitter la salle",
   "more-menu.enter-streamer-mode": "Entrer en mode Streameur",
   "more-menu.exit-streamer-mode": "Sortir du mode Streameur",
-  "more-menu.favorite-room": "Salle Favorite",
-  "more-menu.favorite-rooms": "Salles Favorites",
+  "more-menu.favorite-room": "Salle favorite",
+  "more-menu.favorite-rooms": "Salles favorites",
   "more-menu.help": "Aide",
   "more-menu.invite": "Inviter",
   "more-menu.privacy": "Notes sur la vie privée",
   "more-menu.profile": "Changer de Nom & Avatar",
   "more-menu.report-issue": "Signaler une Erreur",
   "more-menu.room": "Salle",
-  "more-menu.room-info": "Info de la Salle et Réglages",
+  "more-menu.room-info": "Info de la salle et réglages",
   "more-menu.sign-in": "Se connecter",
   "more-menu.sign-out": "Se déconnecter",
   "more-menu.start-tour": "Démarrer la visite guidée",
   "more-menu.support": "Support",
   "more-menu.tos": "Conditions d'utilisation",
   "more-menu.unfavorite-room": "Supprimer la salle des favoris",
   "more-menu.whats-new": "Quoi de neuf",
-  "oauth-screen.description": "Vous devrez vous connecter pour {provider} accéder à la salle.",
+  "new-token-scopes.read-rooms.description": "Lire les données de la salle",
+  "new-token-scopes.write-rooms.description": "Écrire les données de la salle",
```

Comment:
"Lire la description de la salle" and "Écrire la description de la salle" ?

Reply:
In English it's
  "new-token-scopes.read-rooms.description": "Read room data",                  
  "new-token-scopes.write-rooms.description": "Write room data",
I guess it's more than the description of the room. It's all the uploaded files?

Reply:
"Lire les informations liées à la salle" ?

Reply:
Ok slighly better. I changed to "Écrire les informations liées à la salle" as well.